### ISSUE TITLE
Daytime exposure used during night

### ISSUE THREAD

**Issue by dark-nyx** [aaronwmorris/indi-allsky#1085]
For some reason the wrong exposure is used during night:

![Screenshot 2024-01-01 200814](https://github.com/aaronwmorris/indi-allsky/assets/155389682/f03ace23-0406-4192-9d20-d649cbeac086)

Here are the settings used:

![Screenshot 2024-01-01 200937](https://github.com/aaronwmorris/indi-allsky/assets/155389682/fe96eadc-b29f-4a66-a829-87ffc3ca118c)

Any ideas why?

**Comment by aaronwmorris** [comment]
You may need to fully stop the indi-allsky service and start it again to change the min/max exposure settings.

**Comment by dark-nyx** [comment]
So, I did as you suggested. I restarted all 3 services, indiserver, indi-allsky and gunicorn-indi-allsky. As a matter of fact I restarted the whole Server this morning. Still same problem. Going from day to night, the exposures are way too short:

![Screenshot 2024-01-02 185040](https://github.com/aaronwmorris/indi-allsky/assets/155389682/9d3cc3ec-a24e-42a9-9000-4fbe1d3dabbc)

**Comment by aaronwmorris** [comment]
Well, that is really weird.  I am trying to track this down.

**Comment by aaronwmorris** [comment]
Have you tried updating to the latest code?  I have not found any cause for this behavior and all of the tests in my lab work as they should.

**Comment by dark-nyx** [comment]
I've updated to the latest code, restarted everything and let the thing run for 2 nights. No success. The camera gain changes from 100 (day) to moon-mode (800) to night (1600) without a problem. But the exposure remains the same (0.000105s) when switching from day to night.

**Comment by aaronwmorris** [comment]
I think I need some log entries to tell what is going on.  A dump of the config would be helpful as well.  You might want to supply a support dump.

```
source virtualenv/indi-allsky/bin/activate
./config.py dump

./misc/support_info.sh
```

If you are able and comfortable to grant remote access to your system via SSH, I could login and poke around, too.

**Comment by dark-nyx** [comment]
OK, I believe I found the problem. As a matter of fact there is no problem. The issue was my Target ADU configuration. This is how my current configuration looks like, which looks like it works now:

![Screenshot 2024-01-05 180011](https://github.com/aaronwmorris/indi-allsky/assets/155389682/e4bb6613-837a-436c-b0ee-a92432d8a69e)

I reduced the deviation so that it won't overlap between day and night.
